site/src/pages/CreateTemplatePage/CreateTemplateForm.tsx

```diff
@@ -379,44 +382,91 @@ export const CreateTemplateForm: FC<CreateTemplateFormProps> = ({
+                <Stack direction="column" spacing={0.5}>
+                  <Stack
+                    direction="row"
+                    alignItems="center"
+                    spacing={0.5}
+                    className={styles.optionText}
+                  >
+                    <strong>{t("form.fields.allowUsersToCancel")}</strong>
+
+                    <HelpTooltip>
+                      <HelpTooltipText>
+                        {t("form.tooltip.allowUsersToCancel")}
+                      </HelpTooltipText>
+                    </HelpTooltip>
+                  </Stack>
+                  <span className={styles.optionHelperText}>
+                    {t("form.helperText.allowUsersToCancel")}
+                  </span>
                 </Stack>
-                <span className={styles.optionHelperText}>
-                  {t("form.helperText.allowUsersToCancel")}
-                </span>
               </Stack>
-            </Stack>
-          </label>
-        </FormFields>
+            </label>
+          </FormFields>
+          <FormFields>
+            <label htmlFor="allow_everyone_group_access">
+              <Stack direction="row" spacing={1}>
+                <Checkbox
+                  id="allow_everyone_group_access"
+                  name="allow_everyone_group_access"
+                  disabled={isSubmitting || !allowDisableEveryoneAccess}
+                  checked={form.values.allow_everyone_group_access}
+                  onChange={form.handleChange}
+                />
+
+                <Stack direction="column" spacing={0.5}>
+                  <Stack
+                    direction="row"
+                    alignItems="center"
+                    spacing={0.5}
+                    className={styles.optionText}
+                  >
+                    <strong>Allow the everyone to use the template</strong>
+
+                    <HelpTooltip>
+                      <HelpTooltipText>
+                        If unchecked, only users with the &apos;template
+                        admin&apos; and &apos;owner&apos; role can use this
+                        template until the permissions are updated. Navigate to{" "}
+                        <strong>
+                          Templates &gt; Select a template &gt; Settings &gt;
+                          Permissions
+                        </strong>{" "}
+                        to update permissions.
+                      </HelpTooltipText>
+                    </HelpTooltip>
+                  </Stack>
+                  <span className={styles.optionHelperText}>
+                    This setting requires an enterprise license for the&nbsp;
+                    <Link href="https://coder.com/docs/v2/latest/admin/rbac">
+                      &apos;Template RBAC&apos;
+                    </Link>{" "}
+                    feature to customize permissions. Without this license,
+                    unchecking this box prevents template usage by non-admins.
```

Comment:
They can't uncheck the box if they don't have the entitlement though, so this last sentence seems strange

Reply:
Fair. I was trying to word the rational why the checkbox was disabled. I'll just delete the extra sentence